server/BudgetTracker.Business/Validation.cs

```diff
@@ -34,8 +34,12 @@ public static async Task<bool> IsAccountRegistrationDuplicate(string username, U
 
         public static bool IsCreateBudgetRequestValid(CreateBudgetRequestContract arguments)
         {
-            bool isValid = arguments.Name != null &&
-                arguments.SetAmount != 0M;
+            bool isValid = arguments.Name != null;
```

Comment:
Also didn't we have more logic in here around the budget duration?

Reply:
Yeah it's further down in this function.